apply-pr.js

```diff
@@ -142,15 +142,14 @@ function resolveNames() {
     github.user.getFrom({ user: name }, resolve);
   });
 }
-
+console.log(OUTPUT['PR-URL'] + '.patch');
```

Comment:
Sounds good, let's do that if we end up having a significant amount of debug output at some point.